todo_app/data/mongo_items.py

```diff
@@ -6,8 +6,23 @@
 from .session_items import *
 
 
-#Get Connection sTrign for Mongo
+#Get variable to determin if running local development
+LOCAL_ENV = os.environ.get("ENV")
+
+#github details
+if(LOCAL_ENV == "Local"):
+    GITHUB_CLIENT_ID = os.environ.get("OAUTH_ID_L")
+    GITHUB_CLIENT_SECRET = os.environ.get("OAUTH_KEY_L")
+    GITHUB_CALLBACK_URL = os.environ.get("OAUTH_URL_L")
+else:
+    GITHUB_CLIENT_ID = os.environ.get("OAUTH_ID")
+    GITHUB_CLIENT_SECRET = os.environ.get("OAUTH_KEY")
+    GITHUB_CALLBACK_URL = os.environ.get("OAUTH_URL")
```

Comment:
As above, I don't understand why we have this switch, when we could just let them fill in the appropriate variables - potentially just to make a devs life easier when switching? If so, I'd lean towards just commenting/uncommenting blocks in your `.env` file, so that this code can be simplified personally